README.md

```diff
@@ -141,26 +141,14 @@ your debug UI. Or provide a shortcut from your app settings to dump the database
 
 Want to control the log categories in runtime? Use the `SharedPreferencesCategoryFilter` with your `LogCatChannel` and enable updating of it from your debug UI.
 
-## Download
```

Comment:
This is unrelated change. Should keep it out of this PR to make the history cleaner.